packages/dbml-core/src/export/DbmlExporter.js

```diff
@@ -15,8 +15,24 @@ class DbmlExporter {
     return /\s*(\*|\+|-|\([A-Za-z0-9_]+\)|\(\))/g.test(str);
   }
 
-  static escapeNote (str) {
-    return str.replaceAll("'", "\\'")
+  static escapeNote (str, indentLengh) {
+    if (str === null) {
+      return ''
+    }
+    if (str.match(/[\n\r']/)) {
+      const closure = function() {
+        function compute_line(line, lineNumber){
+          var prefix = (lineNumber > 0) ? ' '.repeat(indentLengh) : "";
+          return prefix + line;
+        }
+        return compute_line
+      }
+      const comp_line = closure();
+      const res = str.split(/[\r\n]/).map(comp_line).join("\n");
+      return `'''${res}'''`;
+    } else {
```

Comment:
I don't think we should prefix each new line with additional white-space because it would lead to weird formating, and will make markdown comment invalid as well.